src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -448,4 +446,48 @@ std::unordered_map<std::string, BenchmarkTableInfo> AbstractTableGenerator::_loa
   return table_info_by_name;
 }
 
+std::unordered_map<std::string, BenchmarkTableInfo> AbstractTableGenerator::_load_binary_tables_from_json() {
```

Comment:
Put this into the AbstractTableGenerator.

Reply:
You mean into the StorageManager? It's already in the AbstractTableGenerator.

Reply:
Oops, I'm sorry.